xbmc/guilib/GUIImage.dox

```diff
@@ -72,6 +72,17 @@ important, as `xml` tags are case-sensitive.
 | fadetime      | This specifies a crossfade time that will be used whenever the underlying <b>`<texture>`</b> filename changes. The previous image will be held until the new image is ready, and then they will be crossfaded. This is particularly useful for a large thumbnail image showing the focused item in a list, or for fanart or other large backdrops.
 | background    | For images inside a container, you can specify background="true" to load the textures in a background worker thread. [See here for additional information about background loading](http://kodi.wiki/view/Background_Image_Loader).
 
+For the tags "texture" and "bordertexture", the following attributes are available.
```

Comment:
Can you add `@skinning_v20` revision tag for the infill property?

Reply:
Done.